pkg/backingstore/reconciler.go

```diff
@@ -1305,10 +1312,15 @@ func (r *Reconciler) updatePodTemplate() error {
 		r.PodAgentTemplate.Spec.ImagePullSecrets =
 			[]corev1.LocalObjectReference{*r.NooBaa.Spec.ImagePullSecret}
 	}
+
+	bs_label, ok := r.BackingStore.Labels["backingstore"]
+	if !ok || bs_label == "" {
+		bs_label = "noobaa"
+	}
```

Comment:
```suggestion
	bs_label := r.BackingStore.Labels["backingstore"]
	if bs_label == "" {
		bs_label = "noobaa"
	}
```
Just a small suggestion, I think it will return zero "" value if the label is not defined so we can update it.